### DISCUSSION TITLE
cpp code

### DISCUSSION THREAD

**Discussion by Shailesh93602** [azl397985856/leetcode#581] [General]
hey i can contribute by providing cpp codes for the problems

**Comment by cherie-wuyajun** [comment]
这是来自QQ邮箱的自动回复邮件。您好，邮件已收到，我将会尽快给您回复。

**Comment by lording** [comment]
您好，您的来信已收到，我会尽快给您回复的！

**Comment by fyyzkd** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Reply by Shailesh93602** [reply]
when the holidays are ending?

**Comment by xiaosisong** [comment]
我已经收到了，会尽快查看的，谢谢

**Comment by azl397985856** [comment]
I think you can follow this pr. https://github.com/azl397985856/leetcode/pull/566/files

I hope you know how to create a pull request(pr).

**Comment by lording** [comment]
您好，您的来信已收到，我会尽快给您回复的！
